test/backend/NMLUnitTestSuite.scala

```diff
@@ -54,8 +55,10 @@ class NMLUnitTestSuite @Inject()(nmlParser: NmlParser) extends PlaySpec {
         .parse(
           "",
           new ByteArrayInputStream(array),
-          overwritingDatasetId = None,
-          isTaskUpload = true,
+          SharedParsingParameters(useZipName = false,
+                                  overwritingDatasetId = None,
+                                  userOrganizationId = "testOrganization",
+                                  isTaskUpload = true),
```

Comment:
@fm3 Writing such a test is rather difficult as the nmls for the tests are generated with `NmlWriter().toNmlStream` which requires an organization id. This would mean to rewrite `toNmlStream` which is not preferable as one should always supply the `toNmlStream` with an organization. But writing a copy of `toNmlSteam` that supports an optional organizationId is also not preferable :/

=> I skipped this for now but it should be useful for the future to avoid this regression bug introduced by https://github.com/scalableminds/webknossos/pull/8075

Reply:
what are your thoughts?

Reply:
I’d say this is part of the broader problem of testing the libs and webknossos together. That’s a topic we’ll definitely discuss next year.